nano/node/active_transactions.cpp

```diff
@@ -56,33 +58,34 @@ void nano::active_transactions::confirm_frontiers (nano::transaction const & tra
 		prioritize_frontiers_for_confirmation (transaction_a, is_test_network ? std::chrono::milliseconds (50) : std::chrono::seconds (2));
 
 		size_t elections_count (0);
-		std::unique_lock<std::mutex> lock (mutex);
+		lk.lock ();
 		while (!priority_cementable_frontiers.empty () && !stopped && elections_count < max_elections)
 		{
-			auto cementable_account = *priority_cementable_frontiers.begin ();
-			priority_cementable_frontiers.erase (priority_cementable_frontiers.begin ());
-			lock.unlock ();
+			auto cementable_account = *priority_cementable_frontiers.get<1> ().begin ();
+			priority_cementable_frontiers.get<1> ().erase (priority_cementable_frontiers.get<1> ().begin ());
```

Comment:
2 times get same info `priority_cementable_frontiers.get<1> ().begin ()`

Reply:
Made it a separate variable